### PR TITLE
Cross platform

### PR DESCRIPTION
**Pull request by puengel**
Cross platform npm scripts.

### CODE REVIEW

File: ui/react/webpack.config.js [hunk 1]
```diff
@@ -34,21 +34,19 @@ var configuration = {
...
         test: /\.css$/,
-        loaders: ['style-loader', 'css-loader'],
-        include: __dirname + '/src',
+        use: ['style-loader', 'css-loader'],
         exclude: /node_modules/
       },
       {
         test: /\.(ttf)$/,
-        loader: 'url-loader?limit=100000'
+        use: 'url-loader?limit=100000'
```
Comment (Equanox) [inline ui/react/webpack.config.js]: Add exclude : node_modules

File: ui/react/webpack.config.js [hunk 2]
```diff
@@ -34,21 +34,19 @@ var configuration = {
...
       },
       {
         test : /\.jsx?/,
-        loader : 'babel-loader',
-        include : __dirname + '/src'
+        use : 'babel-loader',
```
Comment (Equanox) [inline ui/react/webpack.config.js:50]: Add exclude : node_modules

File: ui/typescript/webpack.config.js [hunk 3]
```diff
@@ -41,19 +41,13 @@ var configuration = {
...
       {
         test: /\.css$/,
-        loaders: ['style-loader'],
-        include: __dirname + '/src',
+        use: ['style-loader', 'css-loader'],
         exclude: /node_modules/
       },
       {
         test: /\.(ttf)$/,
         loader: 'url-loader?limit=100000'
```
Comment (Equanox) [inline ui/typescript/webpack.config.js]: Add exclude : node_modules

File: ui/typescript/webpack.config.js [hunk 4]
```diff
@@ -41,19 +41,13 @@ var configuration = {
...
       },
-      {
-        test: /\.css$/,
-        loader: 'typings-for-css-modules-loader?modules&namedExport&camelCase',
-        exclude: /node_modules/
-      },
       {
         test: /\.(ts|tsx)$/,
```
Comment (Equanox) [inline ui/typescript/webpack.config.js]: Add exclude : node_modules

**Review by Equanox — CHANGES_REQUESTED** [review]
Fix missing exclude: node_modules

**Comment by puengel** [comment]
This has been tested on windows and ubuntu.

**Comment by Equanox** [comment]
Looks good to me :+1:

**Review by Equanox — APPROVED** [review]

**Pull request merged**